app/components/application_component.rb

```diff
@@ -1,3 +1,8 @@
 class ApplicationComponent < ViewComponent::Base
+  delegate :current_user, to: :helpers
+
+  def can_see_unpublished_items?
+	AccessPolicy.new(current_user).can? :see_unpublished_items
+  end
```

Comment:
You should be able to delegate `can?` and then just do `can? :see_unpublished_items`, I think that's better?

Reply:
But again, the bigger issue is -- why have this helper method at all, can we just do `can? :see_unpublished_items` everywhere we'd call this helper method? As per other PR. 

But actually, ok, I'll put a comment in the ticket as a whole about something more generall....

Reply:
You can't delegate to an object, only to a class. But point taken -- I adjusted the code in the spirit of your idea, which is a good one

Reply:
I think we're misunderstanding each other. Above where it already says `delegate :current_user, to: :helpers`? If you change this to `delegate :current_user, :can?, to: :helpers`, you should then be able to write simply `can? :whatever` in a ViewComponent. No? I'm not sure what you mean by "you can't delegate to an object only to a class", so I maybe missing something

Reply:
You weren't missing something. I was missing something. Great idea- totally works.